docs/server/settings.rst

```diff
@@ -294,20 +298,50 @@ Translation environment configuration settings.
             'HOST': 'localhost',
```

Comment:
I agree on the backward, but actually it does contribute as it makes it clear what the TM does, ``default`` indicates nothing.